CONTRIBUTING.md

```diff
@@ -68,10 +68,19 @@ If needed, reference Foundry's [Troubleshooting Installation](https://github.com
 Then run the following:
 
 ```
-$ yarn
-$ yarn bootstrap
+$ yarn install
+$ yarn build
+$ yarn compile
 ```
 
+After that, go to the `configuration` folder found in the [rust repo](https://github.com/nomad-xyz/rust). Pull whichever config you want from there, and place that inside your config folder in `/packages/deploy/config`.
```

Comment:
rather than copying these from the configuration, the intended progression is via #291, where the latest config is loaded from a URL. The configs should not be committed multiple places. We want a single source of truth

Side note, the contributing doc is for regular contributors, whom we want to discourage from using the deploy script directly. The deploy script is primarily intended for use by the team and by power users performing governance actions. I'd avoid setting it up in `CONTRIBUTING.md`

Reply:
Thanks so much for the feedback. Closing this PR as proposed changes are unnecessary & addressed by #291